client/my-sites/stats/stats-subscribers-page/subscribers-highlight-section.jsx

```diff
@@ -0,0 +1,93 @@
+import { CountComparisonCard, Popover } from '@automattic/components';
+import { Icon, info } from '@wordpress/icons';
+import { translate } from 'i18n-calypso';
+import { useRef, useState } from 'react';
+import './subscribers-highlight-section.scss';
+
+function getSubscriberHighlights() {
+	const highlights = [
+		{
+			id: 1,
+			heading: 'Total email subscribers',
+			count: 1410,
+			previousCount: 1080,
+		},
+		{
+			id: 2,
+			heading: 'Free email subscribers',
+			count: 1340,
+			previousCount: 1034,
+		},
+		{
+			id: 3,
+			heading: 'Paid email subscribers',
+			count: 70,
+			previousCount: 46,
+		},
+		{
+			id: 4,
+			heading: 'WordPress.com subscribers',
+			count: 322,
+			previousCount: 326,
+		},
+	];
+	return highlights;
+}
+
+function SubscriberHighlightsHeader() {
+	const [ isTooltipVisible, setTooltipVisible ] = useState( false );
+	const infoReferenceElement = useRef( null );
+	const localizedTitle = translate( 'All time stats', {
+		comment: 'Heading for Subscribers page highlights section',
+	} );
+	return (
+		<h1 className="highlight-cards-heading">
+			{ localizedTitle }{ ' ' }
+			<span
+				className="info-wrapper"
+				ref={ infoReferenceElement }
+				onMouseEnter={ () => setTooltipVisible( true ) }
+				onMouseLeave={ () => setTooltipVisible( false ) }
+			>
+				<Icon className="info-icon" icon={ info } />
+			</span>
+			<Popover
+				className="tooltip tooltip--darker tooltip-subscribers-page highlight-card-tooltip"
+				isVisible={ isTooltipVisible }
+				position="bottom right"
+				context={ infoReferenceElement.current }
+			>
+				<div className="highlight-card-tooltip-content">
+					<p>Tooltip content goes here.</p>
+				</div>
+			</Popover>
+		</h1>
+	);
+}
+
+function SubscriberHighlightsListing() {
+	const highlights = getSubscriberHighlights();
+
+	return (
+		<div className="highlight-cards-list">
+			{ highlights.map( ( highlight ) => (
+				<CountComparisonCard
+					key={ highlight.id }
```

Comment:
Instead of using `id`, we could simply use the 'heading' as the key.

```suggestion
			{ highlights.map( ( highlight ) => (
				<CountComparisonCard
					key={ highlight.heading }
```

Reply:
I'll push a commit to address this.